include/cbl/CBLTLSIdentity.h

```diff
@@ -0,0 +1,240 @@
+_cbl_warn_unused
+FLSliceResult CBLCert_SubjectNameComponent(CBLCert*, FLString attributeKey) CBLAPI;
+
+/** Returns the time range during which a (signed) certificate is valid.
+    @param cert  The signed certificate.
+    @param outCreated  On return, the date/time the cert became valid (was signed).
+    @param outExpires  On return, the date/time at which the certificate expires. */
+void CBLCert_getValidTimespan(CBLCert* cert,
+                             CBLTimestamp* _cbl_nullable outCreated,
+                             CBLTimestamp* _cbl_nullable outExpires) CBLAPI;
+
+/** Returns a certificate's public key.
+    @note You are responsible for releasing the returned key reference. */
+_cbl_warn_unused
+CBLKeyPair* CBLCert_PublicKey(CBLCert*) CBLAPI;
+
+/** Returns the certificate chain of the given TLS identity. */
+_cbl_warn_unused
+CBLCert* CBLTLSIdentity_Certificates(CBLTLSIdentity*) CBLAPI;
+
+/** Returns the date/time at which the first certificate in the chain expires. */
+CBLTimestamp CBLTLSIdentity_Expiration(CBLTLSIdentity*) CBLAPI;
+
+/**
+ Creates a self-signed TLS identity with the given RSA keypair and certificate attributes.
+ @Note The Common Name (kCBLCertAttrKeyCommonName) attribute is required.
+ @Note You are responsible for releasig the returned reference. */
+_cbl_warn_unused
+CBLTLSIdentity* _cbl_nullable CBLTLSIdentity_SelfSignedCertIdentity(bool server,
+                                                                    CBLKeyPair* keypair,
+                                                                    FLDict attributes,
+                                                                    CBLTimestamp expiration,
+                                                                    CBLError* _cbl_nullable outError) CBLAPI;
+
+#if !defined(__linux__) && !defined(__ANDROID__)
+
+/**
+ Creates a self-signed TLS identity with the given RSA keypair and certificate attributes. If the persietent label is specified,
+ The identity will be persisted in the platform's keystore (Keychain for Apple or x509 Key/Certificate Stores for Windows)
+ @Note The Common Name (kCBLCertAttrKeyCommonName) attribute is required.
+ @Note You are responsible for releasing the returned reference.
+ */
+_cbl_warn_unused
+CBLTLSIdentity* _cbl_nullable CBLTLSIdentity_SelfSignedCertIdentityWithLabel(bool server,
+                                                                    FLString persistentLabel,
+                                                                    FLDict attributes,
+                                                                    CBLTimestamp expiration,
+                                                                    CBLError* _cbl_nullable outError) CBLAPI;
+
+/** Deletes the identity from the platform's keystore (Keychain for Apple or x509 Key/Certificate Stores for Windows) witih the given persistent label. */
+bool CBLTLSIdentity_deleteIdentityWithLabel(FLString persistentLabel,
```

Comment:
`delete` in `CBLTLSIdentity_deleteIdentityWithLabel` is not capitalized.

Reply:
Hey, @blaugold thank you very much for taking the time to do the review
Will capitalize it